sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_base_test.hpp

```diff
@@ -44,7 +44,11 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
         Azure::Core::Http::HttpStatusCode expectedCode = Azure::Core::Http::HttpStatusCode::Ok)
     {
       auto rawResponse = response.ExtractRawResponse();
-      EXPECT_EQ(rawResponse->GetStatusCode(), expectedCode);
+      EXPECT_EQ(
+          static_cast<typename std::underlying_type<Azure::Core::Http::HttpStatusCode>::type>(
+              rawResponse->GetStatusCode()),
+          static_cast<typename std::underlying_type<Azure::Core::Http::HttpStatusCode>::type>(
+              expectedCode));
```

Comment:
What's the benefit of this?

Reply:
when EXPECT_EQ works with enum class, it gets resolved as a binary comparison, then, if values are not equal, gtest will print something like:
```
/mnt/vss/_work/1/s/sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_base_test.hpp:47: Failure
209: Expected equality of these values:
209:   rawResponse->GetStatusCode()
209:     Which is: 4-byte object <94-01 00-00>
209:   expectedCode
209:     Which is: 4-byte object <C8-00 00-00>
```

This is not easy to understand.  Now, after the change, gtest will resolve to int == int and show the real int value behind the enum